config/experiment/sdss_galaxy_encoder.yaml

```
@@ -18,3 +18,6 @@ model:
 optimizer:
   kwargs:
     lr: 1e-4
+
+dataset:
+  batch_size: 8
```

newline missing here 
```suggestion
  batch_size: 8
  
```